SwiftyContextMenu/ContextMenuBackgroundBlurView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ContextMenuBackgroundBlurView.swift
+//  SwiftyContextMenu
+//
+//  Created by Paul Bancarel on 28/11/2020.
+//
+
+final class ContextMenuBackgroundBlurView : IntensityVisualEffectView {
```

Comment:
the coma should be near the type name
`class ContextMenuBackgroundBlurView: IntensityVisualEffectView`

Reply:
👍 I'm really likes that attention to detail